config/settings.yml

```diff
@@ -1355,9 +1355,10 @@ rrd:
       - fake_email
 
 virtual_regional_office:
-  url: http://va.gov
+  url: https://sandbox.lighthouse.va.gov/abd-vro
   token: 9d3868d1-ec15-4889-8002-2bff1b50ba62
   health_assessment_path: health-data-assessment
+  evidence_pdf_path: evidence-pdf
```

Comment:
These settings will need to be updated in the devops repo + the new k8s manifest configmaps for vets-api-server and sidekiq

Reply:
Thanks for this note! For the devops repo, I know the process is to just open a PR to update `/ansible/deployment/config/vets-api/*-settings.local.yml.j2` with the correct values. What is the process to update the k8s manifest configmaps?

Incidentally, RRD is designed to only call out to external services (including VRO) within Sidekiq context.

Reply:
For K8s, you'll need to update the respective [settings-configmap.yaml](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/apps/vets-api-server/dev/templates/settings-configmap.yaml) for [vets-api-server](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/tree/main/apps/vets-api-server) and [sidekiq](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/tree/main/apps/sidekiq)

Reply:
FYI https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/882 and https://github.com/department-of-veterans-affairs/devops/pull/11980 are open